dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/ShellCommandExecutor.java

```diff
@@ -108,7 +108,7 @@ protected void createCommandFileIfNotExists(String execCommand, String commandFi
                     sb.append(taskRequest.getEnvironmentConfig()).append("\n");
                 } else {
                     if (taskRequest.getEnvFile() != null) {
-                        sb.append("source ").append(taskRequest.getEnvFile()).append("\n");
+                        sb.append(". ").append(taskRequest.getEnvFile()).append("\n");
```

Comment:
I would like to use `#!/bin/bash` to replace `#!/bin/sh`, rather than use `.` to replace `source`

Reply:
> I would like to use `#!/bin/bash` to replace `#!/bin/sh`, rather than use `.` to replace `source`

+1

Reply:
no problem

Reply:
Hi @yutianaiqingtian , We would like to release it in version 3.0.1. Could you submit this ASAP since we are ready to release version 3.0.1.